<?xml version='1.0' encoding='utf-8'?>
<launch>
<!-- GAZEBO arguments -->
<arg name="paused" default="true"/>
<arg name="use_sim_time" default="true"/>
<arg name="gui" default="true"/>
<arg name="headless" default="false"/>
<arg name="debug" default="false"/>
<arg name="world_name" value="$(find franka_cal_sim)/worlds/fast.world"/>
<!-- Load joint controller configurations from YAML file to parameter server -->
<!--<rosparam file="$(find cam_cal_sim)/config/panda_control.yaml" command="load"/>-->

<!-- load the controllers -->

<!--<node name="controller_unspawner" pkg="controller_manager" type="unspawner" respawn="false" output="screen" args="joint_state_controller panda_hand_controller panda_arm_controller" />
<node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="true" output="screen" args="joint_state_controller panda_hand_controller panda_arm_controller" />-->
<!--<node name="spawn_urdf" pkg="gazebo_ros" type="spawn_model" args="-param robot_description -Y 3.1415926 -urdf -model panda" output="screen"/>-->

<node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" respawn="false" output="screen" />

<include file="$(find panda_moveit_config)/launch/planning_context.launch">
  <arg name="load_robot_description" value="true"/>
</include>
<include file="$(find panda_moveit_config)/launch/move_group.launch" respawn="false"/>
<node name="joint_state_desired_publisher" pkg="topic_tools" type="relay" args="joint_states joint_states_desired" respawn="false"/>

</launch>
